package.json

```diff
@@ -25,12 +25,16 @@
     "@types/mocha": "^8.2.1",
     "@types/node": "^14.14.35",
     "@typescript-eslint/eslint-plugin": "^4.18.0",
-    "@typescript-eslint/parser": "^4.18.0",
+    "@typescript-eslint/parser": "^4.19.0",
     "chai": "^4.3.4",
     "eslint": "^7.22.0",
     "eslint-config-prettier": "^8.1.0",
     "ts-generator": "^0.1.1",
     "ts-node": "^9.1.1",
     "typescript": "^4.2.3"
+  },
+  "dependencies": {
+    "eslint-plugin-promise": "^4.3.1",
+    "remove": "^0.1.5"
```

Comment:
Where is this used?

Reply:
balls. I tried to remove a package but added the remove package...

Reply:
also those should be in dev

Reply:
fixed. good catch!